### ISSUE TITLE
Query an order by product name

### ISSUE THREAD

**Issue by michaelgold** [orders-squad/orders#8]
**As a** customer
**I need** to search my order by product name
**So that**  I can view the details of one specific order
**Assumptions:** 
* There will be a search bar on the top of all orders
* By input the product name it will return to the certain order

**Acceptance Criteria:**
```
When customer type the name of the product
Then they can find the order information of this product
```

**Comment by michaelgold** [comment]
I suggest rewording this:

As a customer
I need to search orders by product name
So that I can find the details of my previous orders

**Comment by michaelgold** [comment]
I think the effort level here is a 5

**Comment by EagleTongX** [comment]
the effort level is 5

**Comment by 75divisha** [comment]
I think the effort level here is a 2

**Comment by LihengGong** [comment]
Query + display. I agree with the value 5.
